readme.md

```diff
@@ -1,31 +1,61 @@
 # Transportation & Public Works 3-1-1 Reporting
 
-This repo contains python scripts that pull CSV reports from the [City's 3-1-1 system](https://www.austintexas.gov/department/311). These reports contain information on what was reported by the Customer Service Request (CSR) from the public and when/how the department took action to resolve it. These reports are then stored in a City datahub dataset.
+This repo contains scripts that the Austin Transportation and Public Works Department uses for 311 CSR reporting and visualization.
 
-![data flow diagram showing the CSRs retrieved as CSV files then stored in the city datahub.](docs/data_flow.png)
+***
+
+## CSV Reporting 
+
+### CSR to Socrata 
+
+This python script pulls CSV reports from the [City's 3-1-1 system](https://www.austintexas.gov/department/311). These reports contain information on what was reported by the Customer Service Request (CSR) from the public and when/how the department took action to resolve it. These reports are then stored in a City datahub dataset.
 
-## CSRs to Socrata
+![data flow diagram showing the CSRs retrieved as CSV files then stored in the city datahub.](docs/data_flow.png)
 
 `csr_to_socrata.py` processes a report that contains CSR-level data on 3-1-1 service requests. There is one row per service request.
 
-`$ python etl/csr_to_socrata.py`
+`python -m etl.csv_reporting.csr_to_socrata`
 
-## Flex Notes to Socrata
+### Flex Notes to Socrata
 
 "Flex notes" are questions that are asked based on the type of CSR that is submitted. Each CSR can have multiple flex questions/answers. `flex_notes_to_socrata.py` processes a report that contains the flex notes.
 
-`$ python etl/flex_notes_to_socrata.py`
+`python -m etl.csv_reporting.flex_notes_to_socrata`
 
-## Activities to Socrata
+### Activities to Socrata
 
 CSRs typically contain activities which document the steps that city staff have taken to resolve a CSR. `activities_to_socrata.py` processes a csv report that contains activities.
 
-`$ python etl/activities_to_socrata.py`
+`python -m etl.csv_reporting.activities_to_socrata`
+
+***
+
+## Open311
+
+### Open311 to Socrata
+
+This script utilizes the [Austin Open311 website's](https://311.austintexas.gov/) API to download CSR data and store it 
+in a Socrata open data portal dataset. This allows for more frequent updates as compared to the CSV reporting which is 
```

Comment:
CSV? CSR? 

I think that these to acronyms are too close and both in heavy use and both able to modify the same nouns to be used in this README without special attention. I think the time saved in using the acronyms is not worth the confusion. Do you think maybe spelling these out in the readme at least would be helpful?